github-orgs/kubeflow/org.yaml

```diff
@@ -1085,24 +1085,13 @@ orgs:
           pipelines:
```

Comment:
why do we need this team if we have wg-pipelines-team ?

Reply:
Because they maintain the project.

Reply:
The members are almost identical to the `wg-pipeline-leads`: https://github.com/kubeflow/internal-acls/blob/8bfcac01904a25121979a512418026d65ccf0604/github-orgs/kubeflow/org.yaml#L1205-L1210

Shall we just use the `wg-pipelines-leads` team to add maintainers ?

Reply:
I'd rather delete wg-pipelines-leads

Reply:
I'll defer to @HumairAK, but as I understand it the aim is for _all_ pipelines maintainers to be able to cut releases. In terms of how the community for KFP is structured and run -- there isn't really much of an emphasis on WGs. I'd rather we keep both teams _for now_ until the future of WGs are figured out.

Reply:
This list is all `Approvers`. I think we should only give access to root level approvers.

Reply:
Hey folks, please **do not give write access to reviewers**. Only maintainers should have write access to the KFP repo. Maintainers are the "Approvers" [here](https://github.com/kubeflow/pipelines/blob/master/OWNERS#L6-L12).

Reply:
Sure, updated.